src/src/PluginStructs/P092_data_struct.cpp

```diff
@@ -462,7 +462,7 @@ P092_data_struct::~P092_data_struct() {
   }
 }
 
-bool P092_data_struct::init(int8_t pin1) {
+bool P092_data_struct::init(int8_t pin1, int DeviceIndex) {
```

Comment:
Is it possible to use a different name for this variable?
In ESPEasy we also have a DeviceIndex, which means something completely different.

Reply:
I have changed the variable name to P092DeviceIndex (see last commit)